e2e/activity-page.spec.ts

```diff
@@ -7,5 +7,5 @@ testWithII('should display activity page', async ({ page, iiPage }) => {
 
 	await activityPage.waitForReady();
 
-	await expect(page).toHaveScreenshot({ fullPage: true });
```

Comment:
why removing this? don't we do it for all of them the same?

Reply:
I was running the tests on this branch locally and the activity page was throwing random errors. Sometimes the toHaveScreenshot was expecting something that should not be. I then checked what fullPage actually does, and turns out its not necessary unless the screenshot should include "scrollable" content.